### PR TITLE
Add bg and cards for pod members

### PR DESCRIPTION
**Pull request by suhanichawla**

### CODE REVIEW

**Comment by suhanichawla** [comment]
@sachsom95 please review the PR

File: _sass/members.scss [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+background-color: #ffffff;
```
Comment (sachsom95) [inline _sass/members.scss]: probably a bit of nit pick. But could make the style consistent by using 4 spaces?

File: _sass/members.scss [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+	padding: 2rem;
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+	outline: 1px solid transparent;
+	-webkit-perspective: inherit;
+	perspective: inherit;
+	z-index: 2;
+}
+
+
```
Comment (sachsom95) [inline _sass/members.scss]: again small nit picks do try leaving same amount of line gaps

File: _sass/members.scss [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+		width: calc(50% - 2rem);
+	}
+}
+@media screen and (max-width: 32rem) {
+	.col {
+		width: 100%;
+		margin: 0 0 2rem 0;
+	}
+}
+
```
Comment (sachsom95) [inline _sass/members.scss:220]: remove the additional blank lines from end of file, the reason is that the number of lines changed includes the additional gaps and will be reflected in github

**Review by sachsom95 — APPROVED** [review]
All looks good and test as well good job :)

**Comment by sachsom95** [comment]
Merging the first section

**Pull request merged**